charts/curiefense/curiefense/1.0.1+up1.0.0/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+annotations:
+  catalog.cattle.io/certified: partner
+  catalog.cattle.io/display-name: curiefense
+  catalog.cattle.io/release-name: curiefense
+apiVersion: v1
+appVersion: 1.0.0
+description: Complete curiefense deployment
+kubeVersion: '>= 1.19'
+name: curiefense
+version: 1.0.1+up1.0.0
```

Comment:
This chart is missing a logo, which is used in the Rancher UI. Could you please add a logo by setting the `icon` key as following if you have one hosted somewhere?
`icon: https://url-to-your-logo.png`

Alternatively, you can use this repository to host it by dropping the file in `/partner-charts/assets/logos/curiefense.png` and then referencing it here as `https://partner-charts.rancher.io/assets/logos/curiefense.png`.